python2/koans/about_iteration.py

```diff
@@ -47,7 +47,7 @@ def is_even(item):
         seq = [1, 2, 3, 4, 5, 6]
 
         even_numbers = filter(is_even, seq)
-        self.assertEqual(__, even_numbers)
+        self.assertEqual([2, 4, 6], even_numbers)
```

Comment:
In your words, what's the difference between `filter` and `map`?

Reply:
`map` applies the function in all members of the iterator, in contrast, `filter` do a comparison with the members and if that result is true, it's part of the final output.